README.md

```diff
@@ -23,7 +23,7 @@ This method is the easiest to follow, just add the following dependency to your
 <dependency>
     <groupId>com.microsoft.azure.kusto</groupId>
     <artifactId>kusto-ingest</artifactId>
-    <version>2.1.2</version>
+    <version>3.1.3</version>
```

Comment:
Should we put something here that does not require an update per version, e.g. 
``` xml
<version>[current sdk version]</version>, or 
<version>x.y.z</version>
```

Reply:
Good point @yogilad  , Added a badge to represent the version to maven central. Would this be fine

Reply:
The badge is a great idea, but people who copy+paste will now not have the version field.

Maybe add `<version>[see badge below]</version>`

Reply:
Sure @AsafMah , good idea!